peps/pep-0736.rst

```diff
@@ -0,0 +1,312 @@
+sugar. Most of those not in favour of this feature are simply in the camp of 'I
+wouldn't use it'. However over the extensive conversations about this feature,
+the following objections were the most common:
+
+The syntax is ugly
+------------------
+This objection is by far the most common. On the contrary, we argue that:
+
+* This objection is is subjective and many community members disagree.
+* This is already established syntax used in f-strings.
+* Programmers will, as ever, adjust over time.
+
+The feature is confusing
+------------------------
+We argue that:
+
+* Introducing new features typically has this temporary impact.
+* The syntax is very similar to the established ``f'{x=}'`` syntax.
+* The feature and syntax are familiar from other popular modern languages.
+* The expansion of ``x=`` to ``x=x`` is in fact a trivial feature and inherently
+  significantly less complex than ``*arg`` and ``**kwarg`` expansion.
+* This particular syntactic form has been independently proposed on numerous
+  occasions, indicating that it is the most obvious [8]_ [9]_ [13]_.
+
+The feature is not explicit
+---------------------------
+This is based on a misunderstanding of the Zen of Python. Keyword arguments are
+fundamentally more explicit than positional ones where argument assignment is
+only visible at the function definition. On the contrary, the proposed syntactic
+sugar contains all the information as is conveyed by the established keyword
+argument syntax but without the redundancy. Moreover, the introduction of this
+syntactic sugar incentivises use of keyword arguments, making practical Python
+code more explicit.
+
+The feature adds another way of doing things
+--------------------------------------------
+
+The same argument can be made against all syntax changes. This is a simple
+syntactic sugar, much as ``x += 1`` is sugar for ``x = x + 1`` when ``x`` is an
+integer. This isn't tantamount to a 'new way' of passing arguments but a more
+readable notation for the same way.
+
+Renaming the variable in the calling context will break the code
+----------------------------------------------------------------
+
+A ``NameError`` would make the mistake abundantly clear. Moreover, text editors
+could highlight this based on static analysis ‒ ``f(x=)`` is exactly equivalent
+to writing ``f(x=x)``. If ``x`` does not exist, modern editors have no problem
+highlighting the issue.
+
+Recommendations
+===============
+As with any other language feature, the programmer should exercise their own
+judgement about whether to use it in any given context. We do not recommend
+enforcing a rule to use the feature in all cases where it may be applicable.
+
+Reference Implementation
+========================
+`A proposed implementation <https://github.com/Hels15/cpython/tree/last-build>`_
+for cpython has been provided by @Hels15.
+
+References
+==========
+
+.. [1] Issue 36817: Add = to f-strings for easier debugging. - Python tracker
```

Comment:
Prefer putting links and notes inline in the text, instead of at the end.

Reply:
Is you recommendation to replace all references with links? I had aimed to pick whichever seemed most appropriate for the context but I'm happy to comply with whatever the norms are for PEPs.

Reply:
Where appropriate, yes please. I think a footnote is more appropriate if you have something extra to say in the footnote, but otherwise it's preferable to link people directly, no need to make them click twice.

Reply:
Thanks, will update